ui/pages/stats.js

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
```

Comment:
to keep in line with the rest of the codebase 
```suggestion
import React from 'react';
```